### PR TITLE
DOCS-2438 updates to token enforcement policy topic

### PR DESCRIPTION
**Pull request by Mulesoft-Mysti**
Update info about token enforcement policy--clarify and delete repetition that was phrased in different ways, so unclear whether we were describing one thing or two.

### CODE REVIEW

**Comment by Mulesoft-Mysti** [comment]
@sleslie23 , who should do a technical review?

File: api-manager/v/2.x/external-oauth-2.0-token-validation-policy.adoc [hunk 1]
```diff
@@ -1,36 +1,33 @@
 = OAuth 2.0 Access Token Policies
 :keywords: oauth, raml, token, validation, policy
 
-To secure an API in Anypoint Platform using OAuth, using one of the following policies and the matching OAuth provider is recommended. 
+To secure an API in Anypoint Platform using OAuth, we recommend using one of the following policies:
```
Comment (sleslie23) [inline api-manager/v/2.x/external-oauth-2.0-token-validation-policy.adoc]: Can we add this to the style guid if it isn't already?

**Comment by sleslie23** [comment]
LGTM. I'm not sure who the technical reviewer should be. @fermujica or @george-hoffer would know.

File: api-manager/v/2.x/external-oauth-2.0-token-validation-policy.adoc (hunk 1 above)
Comment (Mulesoft-Mysti) [reply api-manager/v/2.x/external-oauth-2.0-token-validation-policy.adoc]: will do.

**Pull request merged**